skt/executable.py

```diff
@@ -654,6 +654,12 @@ def setup_parser():
         type=str,
         help='Use smtp url instead of localhost to send mail',
     )
+    parser_report.add_argument(
```

Comment:
Could we call this --mail-dry-run so that it lines up better with the other --mail arguments?